app/controllers/api/integrations_controller.rb

```diff
@@ -32,15 +33,7 @@ def update
       flash[:notice] = flash_message(:update_success, environment: environment)
       update_mapping_rules_position
 
-      return redirect_to admin_service_integration_path(@service) if apiap?
-
-      if @proxy.send_api_test_request!
-        api_backend = @proxy.api_backend
-        done_step(:api_sandbox_traffic) if api_backend.present? && ApiClassificationService.test(api_backend).real_api?
-        return redirect_to edit_path
-      end
-
-      render :edit
+      redirect_to admin_service_integration_path(@service)
```

Comment:
Since `@service`and `@proxy` are mandatory and found at the beginning of the request... they should be used by `attr_reader :service, :proxy` inside this controller instead of getting it by the instance variable